Source/Core/Core/ConfigManager.cpp

```diff
@@ -325,7 +325,7 @@ void SConfig::SaveNetworkSettings(IniFile& ini)
 
   network->Set("SSLDumpRead", m_SSLDumpRead);
   network->Set("SSLDumpWrite", m_SSLDumpWrite);
-  network->Set("SSLVerifyCert", m_SSLVerifyCert);
+  network->Set("SSLVerifyCertificates", m_SSLVerifyCert);
```

Comment:
The only purpose of renaming it is to reset the default. I don't think we should update the other options just for consistency

I wouldn't update the member variables either, because doing so will force everyone to recompile half of Dolphin.